### PR TITLE
Deploy CF Apps if AppPath is a URL

### PR DESCRIPTION
**Pull request by io-f**
This PR proposes the following changes in the function `CreateAndUploadBitsPackage` to enable deployment of CF Apps with remote `appPath`. 

- If path is a url retrieve the zip file and store in a temp directory.
- Override the path to the location of the zip

### CODE REVIEW

**Comment by io-f** [comment]
Hi @loafoe, Kindly, please review this PR.

File: cloudfoundry/managers/v3appdeployers/standard_strategy_v3.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func (s Standard) Deploy(appDeploy AppDeploy) (AppDeployResponse, error) {
 			return err
 		}
 
-		jobURL, _, err := s.client.DeleteApplication(app.GUID)
+		jobURL, _, _ := s.client.DeleteApplication(app.GUID)
```
Comment (loafoe) [inline cloudfoundry/managers/v3appdeployers/standard_strategy_v3.go:52]: Would this never fail? Is there a way to warn the user in case it still does?

Comment (io-f) [reply cloudfoundry/managers/v3appdeployers/standard_strategy_v3.go]: Fixed.

**Review by loafoe — APPROVED** [review]

**Comment by loafoe** [comment]
LGTM

**Pull request merged**